codecov.yml

```diff
@@ -7,47 +7,42 @@ coverage:
     # Learn more at https://docs.codecov.io/docs/commit-status
     project:
       default:
-        threshold: 1% # allow this much decrease on project
+        target: auto
+        threshold: 0%
+        if_ci_failed: error
+        if_not_found: success
       app:
         target: 70%
-        flags:
-          - app
+        paths:
+          - "app/"
       modules:
         target: 70%
-        flags:
-          - modules
+        paths:
+          - "x/"
+          - "!x/**/client/" # ignore client package
       client:
-        flags:
+        paths:
```

Comment:
let's add a target for the client as well

Reply:
set the target to 70% for client